### DISCUSSION TITLE
How to use the custom 404 page without necessarily setting debug to false

### DISCUSSION THREAD

**Discussion by Grandkojo** [wodoame/expense-tracking-application#16] [General]
What do we have to do?

**Comment by wodoame** [comment]
okay so have you see this line of code in the settings file? 
```python
DEBUG = os.getenv('DEBUG', 'True') == 'True'
```

**Reply by Grandkojo** [reply]
Yeah, I have

**Reply by wodoame** [reply]
This line of code basically looks for the `DEBUG` environment variable in your `.env` file if it can't find it then it'll just use a default value of 'True'. This essentially set's it to 'True' for everyone who has no idea about that variable.

**Reply by wodoame** [reply]
So I wanted you to do something similar to that like this
```python
TEST404 = os.getenv('TEST404', 'False') == 'True'
```
This essentially sets TEST404 to 'False' for everyone.

**Reply by wodoame** [reply]
Then you'll go to the DATABASES section and do this
```python 
if DEBUG or TEST404:
    DATABASES = {
        'default': {
            'ENGINE': 'django.db.backends.sqlite3',
            'NAME': BASE_DIR / os.getenv('DEVELOPMENT_DATABASE', 'db.sqlite3'),
        }
    }
```
So even if DEBUG is False when you enable TEST404 in your env you'll still be able to use the sqlite database.

**Reply by wodoame** [reply]
Any questions?

**Reply by Grandkojo** [reply]
Alright,  I'd try that and get back to ya
